test/node-only/__snapshots__/renderServerResponseBody-node.test.js.snap

```diff
@@ -33,14 +33,14 @@ exports[`renderServerResponseBody allows for strings to be returned from renderA
     <single-spa-router mode="history"
                        base="/"
     >
-      <div id="single-spa-application:undefined">
-      </div>
+      <application name="header">
+      </application>
       <div class="main-content">
         <button>
           Top level content
         </button>
-        <div id="single-spa-application:undefined">
-        </div>
+        <application name="app1">
```

Comment:
Compare to https://github.com/single-spa/single-spa-layout/pull/68/files#diff-9f0ba07bfc2b15cb5078632f6407f395R22